core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/buffer.go

```diff
@@ -150,6 +151,8 @@ type logEventBuffer struct {
 	blocks []fetchedBlock
 	// latestBlock is the latest block number seen
 	latestBlock int64
+	// logsInBuffer is the number of logs currently in the buffer
+	logsInBuffer int64
```

Comment:
You can reduce this var and use AutomationLogsInLogBuffer.Add()

Reply:
Updated